Lib/selectors.py

```diff
@@ -357,7 +357,7 @@ def register(self, fileobj, events, data=None):
             poller_events |= self._EVENT_WRITE
         try:
             self._selector.register(key.fd, poller_events)
-        except Exception:
+        except:
```

Comment:
So you are changing the behaviour here -- now you will intercept BaseExceptions. Is that intended? What if a `SystemExit` occurs and you override it with some exception that `unregister` might rise?